pkgs/ffigen/lib/src/header_parser/parser.dart

```diff
@@ -113,6 +113,21 @@ List<Binding> parseToBindings() {
     tuList.add(tu);
   }
 
+  if (hasSourceErrors) {
+    _logger.warning(
+        "The compiler found some warnings/errors in source files. This might generate invalid bindings due to a wrong compiler guess.");
```

Comment:
-= 'some '

80 chars line limit? You can break the string in two

"The compiler found warnings/errors in source files. " "This will likely generate invalid bindings."

Reply:
done